_includes/macros/share.html

```diff
@@ -31,72 +31,101 @@
    ========================================================================== #}
 
 {% macro share_twitter_url(options) %}
-    {% set share_twitter_url = "http://twitter.com/intent/tweet" %}
-    {% set share_twitter_url = share_twitter_url + "?url=" + request.url|urlencode %}
-    {% set share_twitter_url = share_twitter_url + "&amp;via=CFPB" %}
+    {% set _share_twitter_url = "http://twitter.com/intent/tweet" %}
+    {% set _share_twitter_url = _share_twitter_url + "?url=" + request.url|urlencode %}
+    {% set _share_twitter_url = _share_twitter_url + "&amp;via=CFPB" %}
     {% if options %}
         {# If this is a options object, check for custom fields having been set #}
         {% if options.text %}
-            {% set share_twitter_url = share_twitter_url + "&amp;text=" + options.text|urlencode + "%20--" %}
+            {% set _share_twitter_url = _share_twitter_url + "&amp;text=" + options.text|urlencode + "%20--" %}
         {% endif %}
         {% if options.related %}
-            {% set share_twitter_url = share_twitter_url + "&amp;related=" + options.related %}
+            {% set _share_twitter_url = _share_twitter_url + "&amp;related=" + options.related %}
         {% endif %}
         {% if options.language %}
-            {% set share_twitter_url = share_twitter_url + "&amp;lang=" + options.language %}
+            {% set _share_twitter_url = _share_twitter_url + "&amp;lang=" + options.language %}
         {% endif %}
         {% if options.hashtags %}
-            {% set share_twitter_url = share_twitter_url + "&amp;hashtags=" + options.hashtags %}
+            {% set _share_twitter_url = _share_twitter_url + "&amp;hashtags=" + options.hashtags %}
         {% endif %}
     {% endif %}
-    {{ share_twitter_url }}
+    {{ _share_twitter_url }}
 {% endmacro %}
 
 
 {# ==========================================================================
 
-   share()
+   share.render()
 
    ==========================================================================
 
    Description:
 
-   Make share icons when given:
+   Make share icons when given a dict of options (optional):
 
-   title:                      The title of the thing to share. Used by the
-                               'share via email'.
+       title:           The title of the thing to share. Used by the
+                        'share via email' and 'share via LinkedIn'
```

Comment:
Also very minor: code comment argument options could use consistent use of ending period.